src/config/android/findAndroidAppFolder.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function findAndroidAppFolder(folder) {
```

Comment:
I'll rewrite it l8er today. Don't think comment is needed here. Pretty clear naming imo